packages/Button/__tests__/Button.spec.jsx

```diff
@@ -50,20 +48,10 @@ describe('Button', () => {
   })
 
   describe('responsiveness', () => {
-    it('is full width below medium viewports', () => {
-      mockMatchMedia(767)
-
-      const button = doMount()
-
-      expect(button).toHaveClassName('fullWidth')
-    })
-
-    it('is inline at medium viewports and above', () => {
-      mockMatchMedia(768)
-
+    it('is being assigned responsive css class', () => {
       const button = doMount()
 
-      expect(button).toHaveClassName('inline')
+      expect(button).toHaveClassName('buttonSizing')
```

Comment:
There is no logic here right? I think the `buttonSizing` class is always placed on the element, so the presence of that class is tested through the snapshot test I believe.

I think we could just delete this "responsiveness" test.